pkg/storaged/client.js

```diff
@@ -602,6 +605,53 @@ client.older_than = function older_than(version) {
     return utils.compare_versions(this.manager.Version, version) < 0;
 };
 
+/* Mount users
+ */
+
+client.find_mount_users = (target, is_mounted) => {
+    if (is_mounted === undefined || is_mounted)
+        return python.spawn(mount_users_py, ["users", target], { superuser: "try", err: "message" }).then(JSON.parse);
+    else
+        return Promise.resolve([]);
+};
+
+client.stop_mount_users = (users) => {
+    if (users && users.length > 0) {
+        return python.spawn(mount_users_py, ["stop", JSON.stringify(users)],
+                            { superuser: "try", err: "message" });
+    } else
+        return Promise.resolve();
+};
+
+/* Direct mounting and unmounting
+ *
+ * We don't use UDisks2 for most of our mounting and unmounting in
+ * order to get better control over which entry from fstab is
+ * selected.  Once UDisks2 allows that control, we can switch back to
+ * it.
+ *
+ * But note that these functions still require there to be an entry in
+ * fstab.
+ */
+
+client.mount_at = (block, target) => {
+    const entry = utils.array_find(block.Configuration,
+                                   c => c[0] == "fstab" && utils.decode_filename(c[1].dir.v) == target);
+    if (entry)
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-16991-20220830-075612-e02376d4-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)